lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -90,4 +90,8 @@
     url(r'^generate_bulk_certificate_exceptions', api.generate_bulk_certificate_exceptions,
         name='generate_bulk_certificate_exceptions'),
     url(r'^certificate_invalidation_view/$', api.certificate_invalidation_view, name='certificate_invalidation_view'),
+
+    # rapid response downloands
+    url(r'get_rapid_response_report/(?P<run_id>[^/]*)$', api.get_rapid_response_report,
```

Comment:
Nitpick: I think the URL name should be `rapid_response_report/(?P<run_id>[^/]*)$` (without the `get_`). Just to avoid putting a verb in the resource name